Codes/SlippiReplays/Record.h

```diff
@@ -13,5 +13,9 @@
 #include <Brawl/IT/itManager.h>
 #include "EXIPacket.h"
 #include "brawlback-exi-structures/ExiStructures.h"
+#include <etl/string.h>
```

Comment:
did you mean to add these includes to Record.cpp?
either way, chrono and etl/string should no longer be needed